### PR TITLE
Fix issue #590:  Smart Completions not work on Fx49 

### PR DESCRIPTION
**Pull request by grassofhust**
fix issue #590

### CODE REVIEW

File: common/content/services.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const Services = Module("services", {
         this.jsm = window.Services;
         this.services = {
             "autoCompleteSearch": {
-                class_: "@mozilla.org/autocomplete/search;1?name=history",
+                class_: "@mozilla.org/autocomplete/search;1?name=" + (services.get("vc").compare(VERSION, "49") < 0 ? "history" : "unifiedcomplete"),
```
Comment (MoSal) [inline common/content/services.js]: Vimperator does not work at all in add-on devel builds after applying this commit as-is.
However, changing `history` to `unifiedcomplete` without the version check works.

Comment (gkatsev) [inline common/content/services.js]: Maybe we can do the check outside of this string say before line 17 and store it as a variable we can inline instead?
Might make this more readable as well.

Comment (grassofhust) [inline common/content/services.js]: @MoSal Can you show me what's the `:echo Services.appinfo.platformVersion` output?

@gkatsev Ok. I've updated it.

Comment (MoSal) [inline common/content/services.js]: @grassofhust 

Was `49.0` and now `49.0.1`. But `VERSION` is not the problem. Even when I hard-coded `"49.0.1"`, Vimperator was still broken.

What actually worked is replacing `services.get("vc")` with `this.jsm["vc"]`.

Comment (grassofhust) [inline common/content/services.js]: @MoSal 
Thanks! I've made an update. Can you try it again?

Comment (MoSal) [inline common/content/services.js]: @grassofhust 
This is what I tried already.  
And yes, it works.

Thank you for the original fix.

**Comment by gkatsev** [comment]
Thanks for the PR @grassofhust. Also, for the future, you don't have to keep rebasing your commit, the whole PR will (going forward) just get squashed into a single commit. Would love to see more contributions from you in the future, if you're able and willing!
Thanks for testing this @MoSal!

**Comment by stemar94** [comment]
gentle reminder for merge

**Comment by gkatsev** [comment]
Merged. I'll try and put together a release this week. Would want to see if there are other minor issues that can be fixed. If anyone else can help, that would be great.

**Comment by homemcinza** [comment]
Sorry for the lame question, but is this common/content/services.js file supposed to be on my computer? I couldn't find it.

**Comment by stemar94** [comment]
@homemcinza If you intend to build vimperator on your machine it is, it belongs to the source files.

**Comment by stemar94** [comment]
@homemcinza 

temporary fix(credit goes to: @grassofhust):
Add this line to your .vimperatorrc file:
```
js services.services.autoCompleteSearch.class_="@mozilla.org/autocomplete/search1?name=unifiedcomplete"
```
But you should remove it again, when the new version of vimperator lands.

**Comment by homemcinza** [comment]
@stemar94 Thank you!

**Pull request merged**